components/test-page-setup.js

```diff
@@ -63,16 +109,20 @@ function doTest(pageState, testParams, shaderCode, testState) {
     keys = ["seq", "interleaved", "weak"];
   }
   clearState(testState, keys);
-  const p = runLitmusTest(shaderCode, testParams, pageState.iterations.value, handleResult(testState, keys));
-  p.then(
+  console.log("start a test")
+   await runLitmusTest(shaderCode, testParams, pageState.iterations.value, handleResult(testState, keys)).then(
     success => {
       pageState.running.update(false);
       console.log("success!")
+     
     },
     error => console.log(error)
   );
+  //return 1;
 }
 
+
+
```

Comment:
let's try not to add unnecessary line breaks (nitpicky I know, but you should be able to use vscode to format and get rid of stuff like this)